src/components/DappsGallery/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import styles from './DappsGallery.module.scss';
+import { dappsContent } from './content/DappsGalleryContent';
+
+export default function DappsGallery() {
+  return (
+    <div className={`${styles.dappsContainer} dappsContainer`}>
+      {dappsContent.map((p, index) => (
+        <div key={index} className={`${styles.dappCard} dappCard`}>
+          <a href={p.link} target="_blank" rel="noreferer noopener">
+            <img
+              src={p.project}
+              className={`${styles.dappCardImage} dappCardImage`}
```

Comment:
you only need to use the `styles.dappCardImage` - the extra `dappCardImage` is not needed. You can remove it and it will work. 

Check this to understand how it works:

https://medium.com/@ralph1786/using-css-modules-in-react-app-c2079eadbb87